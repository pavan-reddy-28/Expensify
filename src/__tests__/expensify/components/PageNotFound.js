import React from 'react'
import {shallow } from 'enzyme'
import PageNotFound from '../../../components/NotFoundPage'

test('should render Page not found ', () => {
    const wrapper = shallow(<PageNotFound/>)
    expect(wrapper).toMatchSnapshot();
})
