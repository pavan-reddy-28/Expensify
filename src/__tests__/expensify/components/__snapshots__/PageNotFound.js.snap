// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Page not found  1`] = `
<Fragment>
  <h3>
    404  
    <Link
      to="/"
    >
      Go To DashBoard
    </Link>
  </h3>
</Fragment>
`;
